test/check-argo-status.sh

```diff
@@ -14,45 +14,64 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -ex
+set -e
```

Comment:
This has no effect on making these quieter. The main script has already had `set -ex`.

You can double check from the log: https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/kubeflow_pipelines/2176/kubeflow-pipeline-e2e-test/1175159271583649793#0:build-log.txt%3A714

If you want these to be quiet, you have to be explicit `set +x`, although it will also affect main script env. I think it's fairly complex to make local changes in bash. https://unix.stackexchange.com/a/383581

Do you have a strong need to make this quieter? I don't find it annoying personally and it helps debugging. Therefore, I prefer keeping `set -ex`. I'm interested in your reasoning in making the change.

Reply:
AFAIK most of people in the team were frustrated by hard to read logs for a long time. (Switching to lite deployment really improved the noise problem.). Tracing the script source code does not seem very informative, but adds lots of noise to the test logs. If there is some information not logged right now, we should probably log it instead of tracing everything.

I think it's a good practice where only the user decides whether they want trace or not: `bash -x prog.sh`. If every subprogram script tries to control the logging, they override the user preference.

For this reason I would prefer not to call `set +x` explicitly. I'll rather remove `set -x` from the main script in another PR.

Reply:
I see. totally agree on the reasoning. Let's do this in another PR.